docs/api/covidcast-signals/fb-survey.md

```diff
@@ -411,6 +416,11 @@ believe they need the vaccine. These signals summarize the reasons selected.
 Respondents who do not select any reason (including "Other") are treated as
 missing.
 
+**Note**: After Wave 11 (May 19, 2021), these indicators no longer include
+respondents who indicate in V5d that "I don't believe I need a COVID-19
+vaccine". Item V5d was shown to those who received one dose of a COVID-19
+vaccine, but are not planning to get all recommended doses.
```

Comment:
This is because V5d is discontinued, right? We should say that explicitly here, with an additional sentence saying "Item V5d was removed in Wave 11."